### ISSUE TITLE
bdlink.pw

### ISSUE THREAD

**Issue by Toshiba1127** [Sainan/Universal-Bypass#1592]
Bypass countdown if possible

`https://bdlink.pw/?cupom=MIqVV`

**Comment by Toshiba1127** [comment]
Oops this is Bypass Request, not Broken bypass

**Comment by FireMasterK** [comment]
We can bypass the blog pages by directly navigating to `https://yourtechnology.online/Download/MIqVV`
This is yet another 302 unfortunately.

**Comment by Sainan** [comment]
As you say this is another 302, I suppose this would have the same fix as #1573.

**Comment by FireMasterK** [comment]
That's right, but over here the id of the link is in a parameter here, which requires rewriting the URL, in the other issue the only thing that would be needed is to change the hostname.
